src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div class="multisig-vault-create-review">
+    <h2 class="text-heading-1">
+      {{ $t('modals.createMultisigAccount.title') }}
+    </h2>
+
+    <DetailsItem :label="$t('multisig.creatingAccount')">
+      <template #value>
+        <div class="creator">
+          <Avatar :address="creatorAccount.address" />
+          <div class="creator-data">
+            <BtnPill
+              class="account-select"
+              dense
+            >
+              <FormSelect
+                v-model="creatorAddress"
+                unstyled
+                :default-text="'Choose account'"
+                :options="accountsOptions"
+              />
+            </BtnPill>
+            <AddressTruncated
+              show-explorer-link
+              :address="creatorAccount.address"
+            />
+          </div>
+        </div>
+      </template>
+    </DetailsItem>
+
+    <div class="review-details-row">
+      <DetailsItem :label="$t('multisig.authorizedSigners')">
+        <template #value>
+          <div class="authorized-signers">
+            <div
+              v-for="signer in signers"
+              :key="signer.address"
+              class="authorized-signers-row"
+            >
+              <AccountItem :address="signer.address" />
+              <DialogBox
+                v-if="isLocalAccountAddress(signer.address)"
+                dense
+                v-text="$t('common.you')"
+              />
+            </div>
+          </div>
+        </template>
+      </DetailsItem>
+
+      <DetailsItem
+        label="Consensus"
+        value="3/3 out of 3"
```

Comment:
I think the value shouldn't be static here

Reply:
This is an leftover which is replaced with the slot content. I removed this. Thanks for noticing this.